evm/src/main/java/org/hyperledger/besu/evm/processor/ContractCreationProcessor.java

```diff
@@ -98,7 +98,7 @@ public ContractCreationProcessor(
   private static boolean accountExists(final Account account) {
     // The account exists if it has sent a transaction
     // or already has its code initialized.
-    return account.getNonce() != 0 || !account.getCode().isEmpty();
```

Comment:
I am not understanding why storage should intervene. It isn't mentioned in the Yellow paper AFAICT, it only mentions
- nonce
- code
- balance

and also the request to the state returning $\varnothing$.

<img width="777" alt="image" src="https://github.com/hyperledger/besu/assets/38285177/7584799b-16d9-451d-806c-d58b60768b96">

(source: PARIS VERSION 705168a)

Reply:
Oh, since this is in relation with CREATE's I take it you are verifying the Failure condition F below 

<img width="765" alt="image" src="https://github.com/hyperledger/besu/assets/38285177/841c40d0-c0b6-4c18-988e-ee926107f8f8">

This doesn't include balance, only nonce and code.

Reply:
it's related to this EIP https://eips.ethereum.org/EIPS/eip-7610